lib/poolify.opt.js

```diff
@@ -33,6 +33,7 @@ const poolify = (factory, min, norm, max) => {
     const callback = provide(par);
     if (res) callback(res);
     else delayed.push(callback);
+    return undefined;
```

Comment:
`return pool;`

Reply:
@tshemsedinov there is no `pool` available in this scope.

Reply:
see above